web_archiving/readme.md

```diff
@@ -49,7 +49,7 @@ Capabilities are evaluated as "out of the box" functionality. In cases where a s
 
 * _**WUI**:_ Whether the software offers a web-based user interface.
 
-* _**Library API**:_ Does it provide an explicit libary interface? (I.e., does it provide an API so that you can use it in your own program?)
+* _**Library API**:_ Does it provide an explicit libary interface? (i.e., does it provide an API so that you can use it in your own program?)
```

Comment:
(It is refreshing to be around people who actually care about details like this :-).)

You would indeed use _e.g._ and _i.e._ in lower case form when they introduce text that is part of another sentence. The examples given in that [APA blog posting](http://blog.apastyle.org/apastyle/2012/03/title-case-and-sentence-case-capitalization-in-apa-style.html) are of that kind. However, in the case of our README file, the parenthesized text is a full sentence—the previous sentence ends (with a question mark) and the parenthesized part is a new, separate sentence. Thus, as a separate sentence, the text should begin with a capital letter, even if that text is an abbreviation like e.g. or i.e. This is the difference between

> Was it a strange thing (i.e., was it was weird)?

and

> Was it a strange thing? (I.e., was it weird?)

Another way to see this is to replace i.e. with "that is" and then capitalize accordingly.

> Was it a strange thing (that is, was it was weird)?
> Was it a strange thing? (That is, was it weird?)

My very old paper copy of the APA manual doesn't touch on any of this, and I can't seem to access the Chicago manual online from my institution; the closest I can come to a rule right now is in the [Gregg Reference Manual](http://www.mhhe.com/business/buscom/gregg/toc.php) (I have the 10th edition in paper), where in rule #226, it is stated:

> If the item in parentheses is to be treated as a separate sentence,
> a) The preceding sentence should close with a punctuation mark of its own.
> b) The item in parentheses should begin with a capital letter.

(There's more to #226, but it's about the closing parenthesis and irrelevant to the issue at hand.) No exceptions are given for abbreviations.

The bottom line is that I don't think this is actually an issue of style preferences (and I'm certainly willing to use specific style guides if people prefer). It's about capitalizing the first word of a sentence.

(And if I'm wrong, I'll happily eat my words!)